### DISCUSSION TITLE
Welcome to SwitchBotAPI-BLE Discussions!

### DISCUSSION THREAD

**Discussion by donavanbecker** [OpenWonderLabs/SwitchBotAPI-BLE#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joshtaylor-amway** [comment]
Thanks @donavanbecker for putting this together.  I am attempting to dissect the Temp / Humidity information from a Meter Plus (W2301500).  There seems to be a discrepancy between the documentation and what the device is advertising.  For example, I am seeing the raw data of 0x0201060EFF6909D3C66BD4272E080300973009163DFD690064009730.  When I parse this out, the humidity value seems to be repeated in the temperature value.  Is this a documentation issue or firmware issue on the device?

**Reply by donavanbecker** [reply]
wish I knew. I put this together so that it would be easier to see the commits since it was in a wiki format before. If you figure this out, can you update the [readme](https://github.com/OpenWonderLabs/SwitchBotAPI-BLE/blob/latest/README.md)? and then also update [node-switchbot](https://github.com/OpenWonderLabs/node-switchbot)?

**Comment by Trey-Butte** [comment]
Hey would it be possible to get some sort of example code as I'm really seeming to struggle to set up any communication. I don't really even know where to begin.

**Reply by donavanbecker** [reply]
Look through Homebridge-SwitchBot plugin.

Also readme of node-SwitchBot has examples.
